tests/view/observer/fakeselectionobserver.js

```diff
@@ -100,6 +100,60 @@ describe( 'FakeSelectionObserver', () => {
 		);
 	} );
 
+	it( 'should fire `selectionChangeDone` event after selection stop changing', ( done ) => {
+		// Note that it's difficult to test lodash#debounce with sinon fake timers.
+		// See: https://github.com/lodash/lodash/issues/304
+
+		const spy = sinon.spy();
+
+		viewDocument.on( 'selectionChangeDone', spy );
+
+		// Change selection.
+		changeFakeSelectionPressing( keyCodes.arrowdown );
+
+		// Wait 100ms.
+		setTimeout( () => {
```

Comment:
This will be more complicated, because `beforeEach` uses `setTimeout()`
```javascript
// Ensure selectionchange will not be fired.
setTimeout( () => done(), 100 );
```
I tried to use fake timers instead of this initial `setTimeout` but it didn't work. Besides, `setTimeout` is used few times in other test cases of this class and I'm not sure what is the reason, maybe fake timers are problematic for testing SelectionObserver? This may require deeper investigation.